app/case/views/templates.py

```diff
@@ -66,6 +67,14 @@ def template_email_search_view(request):
 
     return Response(data=EmailTemplateSerializer(templates, many=True).data)
 
+@router.use_route("email-delete")
+@coordinator_or_better_required
+@require_http_methods(["DELETE"])
+def template_email_delete_view(request, pk):
+    template = EmailTemplate.objects.get(pk=pk)
+    template.delete()
+    return render_react_page(request, "Email Templates", "email-template-list", {})
```

Comment:
This should not return `render_react_page` -> which returns a HTML document
Should return a HttpResponse
will discuss further in Slack